### PR TITLE
Update project management page to point to extension pack

### PR DESCRIPTION
**Pull request by nickzhums**

### CODE REVIEW

**Review by jdneo — APPROVED** [review]

File: docs/java/java-project.md [hunk 1]
```diff
@@ -21,7 +21,9 @@ If you run into any issues when using the features below, you can contact us by
 
 Managing a project in VS Code requires the [Project Manager for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-dependency) extension. The extension helps manage class paths and dependencies, and create new projects, packages, and classes.
 
-<a class="tutorial-install-extension-btn" href="vscode:extension/vscjava.vscode-java-dependency">Install the Project Manager for Java</a>
+<a class="tutorial-install-extension-btn" href="vscode:extension/vscjava.vscode-java-pack">Install the Java Extension Pack</a>
```
Comment (Eskibear) [inline docs/java/java-project.md:24]: Do we use the name `Extension Pack for Java` or `Java Extension Pack`?

Comment (nickzhums) [reply docs/java/java-project.md:24]: good point - let us use Extension Pack for Java as it is official name on marketplace

**Review by Eskibear — APPROVED** [review]
LGTM.

**Comment by Eskibear** [comment]
LGTM, "Extension Pack for Java" is the official name.

**Comment by nickzhums** [comment]
FYI - I changed all "Java Extension Pack“ to "Extension Pack for Java"

**Comment by gregvanl** [comment]
Thanks!

**Pull request merged**